airbyte-integrations/connectors/source-looker/acceptance-test-config.yml

```diff
@@ -3,7 +3,7 @@
 connector_image: airbyte/source-looker:dev
 tests:
   spec:
-    - spec_path: "source_looker/spec.json"
+    - spec_path: "source_looker/spec.yaml"
```

Comment:
Is there still really a spec file? It looks like the spec is inlined, how does this work?